packages/schema/src/config/vite.ts

```diff
@@ -40,6 +40,14 @@ export default defineUntypedSchema({
         compilerOptions: {
           $resolve: async (val, get) => val ?? (await get('vue')).compilerOptions
         }
+      },
+      script: {
+        propsDestructure: {
+          $resolve: async (val, get) => val ?? Boolean((await get('vue')).propsDestructure),
+        },
+        defineModel: {
+          $resolve: async (val, get) => val ?? Boolean((await get('vue')).defineModel),
+        },
```

Comment:
it would also be nice if we could update `vue` types as that would then provide a better location to set these options consistently:

https://github.com/nuxt/nuxt/blob/63cf6a5d65e9742c5794c3904574a87fe000cf2c/packages/schema/src/config/app.ts#L6-L23

Reply:
On it

Reply:
What do you think about the description "Vue Experimental: Enable reactive destructure for `defineProps`"?

Didn't find vue documentation to be linked

Reply:
That sounds good to me. Maybe we could link the blog post: https://blog.vuejs.org/posts/vue-3-3

Reply:
RFC it is 👍🏻